### PR TITLE
[4.0.0] remove java 8 from QSG

### PR DESCRIPTION
**Pull request by isharac**

### CODE REVIEW

**Comment by samgnaniah** [comment]
This is quite a high impact change. Affects these pages as well:

https://apim.docs.wso2.com/en/latest/install-and-setup/install/installing-the-product/installing-api-m-runtime/
https://apim.docs.wso2.com/en/latest/install-and-setup/install/installing-the-product/installing-mi/
https://apim.docs.wso2.com/en/latest/install-and-setup/install/installing-the-product/installing-si/
https://apim.docs.wso2.com/en/latest/install-and-setup/install/installing-the-product/installing-mi-dashboard/
https://apim.docs.wso2.com/en/latest/install-and-setup/setup/reference/product-compatibility/#tested-operating-systems-and-jdks

File: en/docs/get-started/streaming-quick-start-guide.md [hunk 1]
```diff
@@ -36,7 +37,12 @@ Let's create a simple Siddhi application that reads data from a XML file, does a
     !!! info
         In this example, the file is located in the `/Users/foo` directory.
 
-2. Navigate to `http://localhost:9390/editor`, click on `New`, and copy and paste the content given below.
+2. Navigate to `http://localhost:9390/editor`, 
+
+    !!! Tip
+        Use `admin` as the username and password.
+        
+click on `New`, and copy and paste the content given below.
```
Comment (samgnaniah) [inline en/docs/get-started/streaming-quick-start-guide.md:45]:
```suggestion
Click on `New`, and copy and paste the content given below.
```
Shall we add this as a numbered step?

File: en/docs/install-and-setup/install/installing-the-product/installing-api-m-runtime.md [hunk 2]
```diff
@@ -22,15 +22,15 @@ You must set your `JAVA_HOME` environment variable to point to the directory whe
...
 
         ``` java
         On Linux:
-        export JAVA_HOME=/usr/java/jdk1.8.0_xx
+        export JAVA_HOME=/usr/java/jdk-11.0.x
         export PATH=${JAVA_HOME}/bin:${PATH}
              
         On OS X:
-        export JAVA_HOME=/System/Library/Java/JavaVirtualMachines/1.8.0.jdk/Contents/Home
+        export JAVA_HOME=/System/Library/Java/JavaVirtualMachines/11.0.x.jdk/Contents/Home
```
Comment (samgnaniah) [inline en/docs/install-and-setup/install/installing-the-product/installing-api-m-runtime.md:33]: This didn't work for me actually. Did you test this?

Comment (isharac) [reply en/docs/install-and-setup/install/installing-the-product/installing-api-m-runtime.md:33]: you have to use the installed JDK version.

File: en/docs/get-started/streaming-quick-start-guide.md (hunk 1 above)
Comment (isharac) [reply en/docs/get-started/streaming-quick-start-guide.md]: fixed

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**